ORK1Kit/ORK1Kit/Common/CEVRK1Theme.m

```diff
@@ -13,30 +13,58 @@
 #import "ORK1ContinueButton.h"
 #import "ORK1TaskViewController.h"
 #import "ORK1Task.h"
+#import "ORK1Step.h"
+
+@interface UIColor (LightAndDark)
+- (UIColor *)lighterColor;
+- (UIColor *)darkerColor;
+@end
 
 NSNotificationName const CEVORK1StepViewControllerViewWillAppearNotification = @"CEVORK1StepViewControllerViewWillAppearNotification";
+NSString *const CEVThemeAttributeName = @"CEVThemeAttributeName";
 NSString *const CEVRK1ThemeKey = @"cev_theme";
 
-@interface CEVRK1Theme()
-@property (nonatomic, assign) CEVRK1ThemeType themeType;
+@implementation CEVRK1Gradient
+
+- (instancetype)init {
+    if ((self = [super init])) {
+        self.direction = CEVRK1GradientDirectionTopToBottom;
+        self.startColor = [UIColor clearColor];
+        self.endColor = [UIColor clearColor];
+    }
+    return self;
+}
+
 @end
 
 @implementation CEVRK1Theme
```

Comment:
I'm getting an error that this does not conform to NSCopying?

Reply:
I've removed the NSCopying declaration.